### DISCUSSION TITLE
Great job!

### DISCUSSION THREAD

**Discussion by imartinez0101** [mkasberg/ghostty-ubuntu#6] [General]
I have tested it in Debian  trixie/sid  with Cinnamon Desktop and it works perfectly.

**Comment by mkasberg** [comment]
Thanks for letting me know!

**Comment by kamruzzaman05** [comment]
> I have tested it in Debian trixie/sid with Cinnamon Desktop and it works perfectly.

Perfect

**Reply by mkasberg** [reply]
Thanks for the note! Glad it's working for you!

**Comment by patterns** [comment]
Also chiming in, thanks! 1.1.2 and now 1.1.3 working on raspberry pi 4 (Debian Bookworm).
